recipes/nanomsg/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake
```

Comment:
The test_v1_package is no longer mandatory for new recipes in CCI. You can remove it if you want: https://github.com/conan-io/conan-center-index/discussions/17406

Reply:
OK，I will remove it to promote the use of new generators in my team.